packages/database/prisma/schema.prisma

```diff
@@ -326,3 +327,14 @@ model PurchaseUserTransfer {
   confirmedAt   DateTime?
   completedAt   DateTime?
 }
+
+
+model Bookmark {
+  id            String                    @id @default(uuid())
+  userId        String 
+  user          User                      @relation(fields: [userId], references: [id], onDelete: Cascade)
+  resourceId    String
+  type          String
+  fields        Json                      @default("{}")
+  createdAt     DateTime                  @default(now())
+}
```

Comment:
prisma schema model now more generic

Reply:
should always have an updated and deleted at column imo